### PR TITLE
Add support for Elasticsearch Rate Metric Aggregator

### PR DESCRIPTION
**Pull request by jimbolimbo13**
## What does this do?

This adds support for the "Rate Aggregation" as a metric in an Elasticsearch query.

image

## Why is it a good idea?

This is supported in Grafana, and my team needed it for something. I figure others might too.

### CODE REVIEW

File: grafanalib/elasticsearch.py [hunk 1]
```diff
@@ -498,3 +498,48 @@ def to_json_data(self):
             'inlineScript': self.inline,
             'settings': self.settings,
         }
+
```
Comment (JamesGibo) [inline grafanalib/elasticsearch.py:501]:
```suggestion


```

Comment (JamesGibo) [reply grafanalib/elasticsearch.py:501]: Fix linting error

**Review by JamesGibo — CHANGES_REQUESTED** [review]
Thanks for the PR, it looks good, just a minor import missing.

Add the following to fix the test error
```
from attr.validators import in_
```

Please could you also add a note to the release notes?

I have also added an issue to test the Elasticsearch file as realised it is missing a test file. 
https://github.com/weaveworks/grafanalib/issues/589

**Review by JamesGibo — APPROVED** [review]
LGTM

**Comment by jimbolimbo13** [comment]
haha, you were too quick. I just pushed the changelog change, but it was missed

**Pull request merged**